client/lib/domains/resolve-domain-status.js

```diff
@@ -49,6 +49,8 @@ export function resolveDomainStatus(
 					statusText: translate( 'Action required' ),
 					statusClass: 'status-error',
 					icon: 'info',
+					listStatusText: translate( 'Complete setup' ),
+					listStatusClass: 'warning',
```

Comment:
I'm wondering if we should change this colour to the red so that it matches the colour on the domain settings screen. What do you think?

Reply:
I was going to ask you about this - do you think the severity of this is error or warning (mapped domain not pointing to us)? I'd like to be consistent but just wonder if we should go for warning instead of error in both places?

Reply:
I was bouncing back and forth on this but I can go with warning. Can we update the copy to match as well so it says complete setup on the settings page as well?